agouti.go

```diff
@@ -43,6 +43,22 @@ func ChromeDriver(options ...Option) *WebDriver {
 	return NewWebDriver("http://{{.Address}}", command, options...)
 }
 
+// EdgeDriver returns an instance of a EdgeDriver WebDriver.
+//
+// Provided Options will apply as default arguments for new pages.
+// New pages will accept invalid SSL certificates by default. This
+// may be disabled using the RejectInvalidSSL Option.
+func EdgeDriver(options ...Option) *WebDriver {
+	var binaryName string
+	if runtime.GOOS == "windows" {
```

Comment:
Let's fail more gracefully on non-Windows OSs here. The method could return `nil`, for instance.